XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -222,6 +236,14 @@ private void EnabledRow(Host host, UpdateType type, int index)
                     return;
                 }
 
+                
+                if (poolOfOne != null && PoolForbidsAutomaticUpdates(poolOfOne))

```

Comment:
Also, having it here, this feature can still be used to see if the pool is up-to-date at least.